mobile/src/reducers/organization/organization.action.ts

```diff
@@ -37,12 +37,6 @@ export interface LoadEmployeesForDepartment extends Action {
 
 export const loadEmployeesForDepartment = (departmentId: string): LoadEmployeesForDepartment => ({ type: 'LOAD_EMPLOYEES_FOR_DEPARTMENT', departmentId });
 
-export interface ErrorLoadFailed extends Action {
-    type: 'ERROR-LOAD-FAILED';
-    errorMessage: string;
-}
-export const errorloadFailed = (errorMessage: string): ErrorLoadFailed =>  ({ type: 'ERROR-LOAD-FAILED', errorMessage });
```

Comment:
It should be used. And move it to the `error-load-failed.action.ts` and use for:
```
...
.catch(e => Observable.of(errorLoadFailed(e.message)))
```